src/libcore/ptr.rs

```diff
@@ -74,7 +74,10 @@ pub const fn null_mut<T>() -> *mut T { 0 as *mut T }
 ///
 /// # Safety
 ///
-/// This is only unsafe because it accepts a raw pointer.
+/// This function copies the memory through the raw pointers passed to it
+/// as arguments, which means they will be dereferenced.
```

Comment:
This is already defined at the language level; repeating it here feels extraneous.